### PR TITLE
Update README with config for MinIO

### LINKED ISSUE #170: Add Example to support with MinIO Configuration
**Title:** Add Example to support withMinIO Configuration

**Description:**

The current README is great for setting up with standard AWS s3 configurations. 
However, including an example for users who try to use `litdata` with MinIO would be beneficial.

**Proposed Addition:**

Add a section to the README to demonstrate how to configure `litdata` to work with MinIO. Below is a proposed example:
The content of the configuration file should be as follows:

```ini
[default]
aws_access_key_id = ACCESS_KEY_HERE
aws_secret_access_key = SECRET_KEY_HERE
endpoint_url = https://minio.your-domain.com # self deployed minio url
```

### PR DESCRIPTION
**Pull request by bhimrazy**
Adds Example to support with MinIO Configuration

Fixes #170.

Here is an example repo: https://github.com/bhimrazy/litdata-with-minio 
to validate the configurations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,37 @@ The `optimize` operator supports any data structures and types. Serialize whatev
 
 Cloud providers such as [AWS](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), [Google Cloud](https://cloud.google.com/storage/docs/uploading-objects?hl=en#upload-object-cli), [Azure](https://learn.microsoft.com/en-us/azure/import-export/storage-import-export-data-to-files?tabs=azure-portal-preview) provide command line clients to upload your data to their storage solutions.
 
+If you're using [MinIO](https://min.io/), an S3-compatible object storage server,
```
Comment (tchaton) [inline README.md:93]: Let's put this at the end of key features. This is bloating the explanation.

Comment (tchaton) [reply README.md:93]: And let s put the link to your repo too

Comment (bhimrazy) [reply README.md:93]: Sure

**Review by tchaton — APPROVED** [review]
Thanks :)

**Pull request merged**